playwright/_impl/_transport.py

```diff
@@ -96,3 +96,6 @@ def send(self, message: Dict) -> None:
         self._output.write(
             len(data).to_bytes(4, byteorder="little", signed=False) + data
         )
+
+    def __repr__(self) -> str:
+        return f"<Transport running={not self._stopped} executable={str(self._driver_executable)}>"
```

Comment:
```suggestion
```
Its not exposed, so its not needed afaik.

Reply:
Yeah, I was waiting for feedback so as to continue for tests